CONTRIBUTING.md

```
@@ -0,0 +1,178 @@
+**Requirements**: JDK 17+, Android SDK API 35+
+
+### 3. Create a Branch
+
+```bash
+git checkout -b feature/your-feature-name
+# or
+git checkout -b fix/your-fix-name
+```
+
+Use one of these branch prefixes:
+| Prefix | Purpose |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)